### DISCUSSION TITLE
Assignment of the work on the process layer and system layer

### DISCUSSION THREAD

**Discussion by HeinrichPet** [International-Data-Spaces-Association/IDS-RAM_4_0#62] [General]
Assignment of the work on the process layer and system layer.

Standards for Process Layer:
- Processes should be represented as sequence diagrams between components as a figure via draw.io (please also upload the files)
- Sequence diagrams needs to be descriped with text
- No BPMN, cause BPMN is used in Business Layer / Rule Book

Standards for System Layer:
- Do not name technologies #27 
- Describe in an abstract way, which components/functionalities are needed for the IDS component
- Describe the deployment

**Comment by HeinrichPet** [comment]
# Identity Provider
### Sections to work on
- [3.3.1 Onboarding](https://github.com/International-Data-Spaces-Association/IDS-RAM_4_0/blob/creating_new_structure/documentation/3_Layers_of_the_Reference_Architecture_Model/3_3_Process_Layer/3_3_1_Onboarding.md) 
-  [3.5.1 Indentity Provider](https://github.com/International-Data-Spaces-Association/IDS-RAM_4_0/blob/creating_new_structure/documentation/3_Layers_of_the_Reference_Architecture_Model/3_5_System_Layer/3_5_1_Identity_Provider.md)                      

### Related Issues
- #59 
- #57

### Maintainer
@gbrost can you or someone from AISEC write this sections?

**Comment by HeinrichPet** [comment]
# Connector
### Related sections
- [3.3.2 Data Offering](https://github.com/International-Data-Spaces-Association/IDS-RAM_4_0/tree/creating_new_structure/documentation/3_Layers_of_the_Reference_Architecture_Model/3_3_Process_Layer/3_3_2_Data_Offering.md)
- [3.3.3 Contract Negotiation](https://github.com/International-Data-Spaces-Association/IDS-RAM_4_0/tree/creating_new_structure/documentation/3_Layers_of_the_Reference_Architecture_Model/3_3_Process_Layer/3_3_3_Contract_Negotiation.md)
- [3.3.4 Exchanging Data](https://github.com/International-Data-Spaces-Association/IDS-RAM_4_0/tree/creating_new_structure/documentation/3_Layers_of_the_Reference_Architecture_Model/3_3_Process_Layer/3_3_4_Exchanging_Data.md)
- [3.5.2 Connector](https://github.com/International-Data-Spaces-Association/IDS-RAM_4_0/blob/creating_new_structure/documentation/3_Layers_of_the_Reference_Architecture_Model/3_5_System_Layer/3_5_2_0_Connector.md)              

### Related Issues
- #16 
- #21 (Data Offering / Negotiation)
- #58 
- #57 (for 3.3.3)
- #31 (for 3.3.3) 
- #28 (for 3.3.4)
 
### Maintainer
@tmberthold can you write a first draft for [3.3.2 Data Offering](3_3_2_Data_Offering.md) (see also [IDS-G-Pre/connector-interaction](https://github.com/International-Data-Spaces-Association/IDS-G-pre/tree/connector-interaction/Communication/sequence-diagrams/data-connector-to-data-connector#connector-self-description-request))
@juliapampus can you write a first draft for [3.3.3 Contract Negotiation](3_3_3_Contract_Negotiation.md) & [3.3.4 Exchanging Data](3_3_4_Exchanging_Data.md) 
@HeinrichPet will start a first draft for [3.5.2 Connector](https://github.com/International-Data-Spaces-Association/IDS-RAM_4_0/blob/creating_new_structure/documentation/3_Layers_of_the_Reference_Architecture_Model/3_5_System_Layer/3_5_2_0_Connector.md)

**Reply by HeinrichPet** [reply]
@tmberthold maybe you do not have to write the Broker section, see [discussion below](https://github.com/International-Data-Spaces-Association/IDS-RAM_4_0/discussions/62#discussioncomment-1814129)

**Comment by HeinrichPet** [comment]
# App Store
### Related sections
- [3.3.5 Publishing and using Data Apps](https://github.com/International-Data-Spaces-Association/IDS-RAM_4_0/tree/main/documentation/3_Layers_of_the_Reference_Architecture_Model/3_3_Process_Layer/3_3_5_Publishing_and_using_Data_Apps.md)  
- [3.5.3 App Store and Data Apps](https://github.com/International-Data-Spaces-Association/IDS-RAM_4_0/tree/main/documentation/3_Layers_of_the_Reference_Architecture_Model/3_5_System_Layer/3_5_3_App_Store_and_Data_Apps.md )        

### Related Issues
- #14
- #40 

### Maintainer
@ahemaid can you write a first draft for both chapters? There are many parts you can directly take from [IDS Think Tank](https://github.com/International-Data-Spaces-Association/IDS-ThinkTank/tree/data-apps/ids-data-apps). However, keep in mind that the RAM is technologie agnostic and only descripe in an abstract way how the App Store and Apps works.

**Comment by HeinrichPet** [comment]
# Broker
### Related sections
- [3.3.2 Data Offering / Metadata Broker](https://github.com/International-Data-Spaces-Association/IDS-RAM_4_0/blob/creating_new_structure/documentation/3_Layers_of_the_Reference_Architecture_Model/3_3_Process_Layer/3_3_2_Data_Offering.md#metadata-broker-interaction)
- [3.5.4 Broker](https://github.com/International-Data-Spaces-Association/IDS-RAM_4_0/blob/creating_new_structure/documentation/3_Layers_of_the_Reference_Architecture_Model/3_5_System_Layer/3_5_4_Broker.md)

### Related Issues
- #15 
- #25 
- #22
- #21

### Maintainer
@sebbader can you, or someone from IAIS, write a first draft for the two sections?

**Comment by HeinrichPet** [comment]
# Clearing House
### Related sections
- [3.3.3 Contract Negotiation](https://github.com/International-Data-Spaces-Association/IDS-RAM_4_0/tree/creating_new_structure/documentation/3_Layers_of_the_Reference_Architecture_Model/3_3_Process_Layer/3_3_3_Contract_Negotiation.md) (partly)
- [3.3.6 Policy Enforcement](https://github.com/International-Data-Spaces-Association/IDS-RAM_4_0/tree/creating_new_structure/documentation/3_Layers_of_the_Reference_Architecture_Model/3_3_Process_Layer/3_3_6_Policy_Enforcement.md) (partly)
- [3.5.5 Clearing House](https://github.com/International-Data-Spaces-Association/IDS-RAM_4_0/blob/creating_new_structure/documentation/3_Layers_of_the_Reference_Architecture_Model/3_5_System_Layer/3_5_5_Clearing_House.md)

### Related Issues
- #13 
- #22 
- #55 

### Maintainer
@kragall can you write a first draft at least for the 3.5.5 Clearing House section?

**Reply by kragall** [reply]
I've created a first draft  for the Clearing House section in #85

**Reply by kragall** [reply]
Clearing House is also relevant in section [3.3.4 Exchanging Data](https://github.com/International-Data-Spaces-Association/IDS-RAM_4_0/blob/main/documentation/3_Layers_of_the_Reference_Architecture_Model/3_3_Process_Layer/3_3_4_Exchanging_Data.md). Its part of the _Invode Data Operation_-process

**Comment by HeinrichPet** [comment]
# Vocabulary Hub
### Related sections
- [3.5.6 Vocabulary Hub](https://github.com/International-Data-Spaces-Association/IDS-RAM_4_0/blob/creating_new_structure/documentation/3_Layers_of_the_Reference_Architecture_Model/3_5_System_Layer/3_5_6_Vocabulary_Hub.md)          
- Missing chapter in Process Layer

### Related Issues

### Maintainer
@sebbader can you, or someone from IAIS, write a first draft in [3.5.6 Vocabulary Hub](https://github.com/International-Data-Spaces-Association/IDS-RAM_4_0/blob/creating_new_structure/documentation/3_Layers_of_the_Reference_Architecture_Model/3_5_System_Layer/3_5_6_Vocabulary_Hub.md)    and maybe also a first draft in a new chapter of the process layer?

**Comment by HeinrichPet** [comment]
# Policy Enforcement
### Related sections
- [3.3.6 Policy Enforcement](https://github.com/International-Data-Spaces-Association/IDS-RAM_4_0/blob/creating_new_structure/documentation/3_Layers_of_the_Reference_Architecture_Model/3_3_Process_Layer/3_3_6_Policy_Enforcement.md)          
- Missing chapter in System Layer?

### Related Issues
#27 

### Maintainer
@hosseinzadeha & @Brandstaedter can you, or someone from IESE, write a first draft in [[3.3.6 Policy Enforcement](https://github.com/International-Data-Spaces-Association/IDS-RAM_4_0/blob/creating_new_structure/documentation/3_Layers_of_the_Reference_Architecture_Model/3_3_Process_Layer/3_3_6_Policy_Enforcement.md) for the related processes?
Do we need a seperate chapter in the system layer?
